src/store/modules/layers.store.js

```diff
@@ -395,6 +400,7 @@ const actions = {
      * @param {string} dispatcher Action dispatcher name
      */
     updateLayer({ commit, getters }, { index, layer, dispatcher }) {
+        console.log('updateLayer', index, layer.name)
```

Comment:
You can check your console in the browser and make sure you've activated the "Verbose" level, this information is already present (each mutation and actions are logged, with they payload etc...)